CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## 0.2.1-dev2
+## 0.2.2
```

Comment:
I changed the version from a dev version to a release version and rolled the enhancements from the latest dev version into this new release version. Is that ok?

Reply:
Can we keep this dev so I can coordinate when the next release goes out?